app/models/manageiq/providers/openstack/network_manager/event_target_parser.rb

```diff
@@ -34,7 +34,8 @@ def parse_ems_event_targets(ems_event)
                     :cloud_networks
                   elsif ems_event.event_type.start_with?("subnet.")
                     :cloud_subnets
-                  elsif ems_event.event_type.start_with?("security_group.")
+                  elsif ems_event.event_type.start_with?("security_group")
```

Comment:
Minor but I'd prefer being explicit e.g. `event_type.start_with?("security_group.") || event_type.start_with?("security_group_rules..")` just to prevent possibly hitting on something unintended

Reply:
Yes, updated.